deploy-web/src/pages/providers/[owner]/index.tsx

```diff
@@ -18,6 +18,7 @@ import { CustomNextSeo } from "@src/components/shared/CustomNextSeo";
 import { UrlService } from "@src/utils/urlUtils";
 import { getNetworkBaseApiUrl } from "@src/utils/constants";
 import axios from "axios";
+import { differenceInMinutes, sub } from "date-fns";
```

Comment:
date fns supports tree shaking but to make use of it I think functions need to be imported one by one
```suggestion
import differenceInMinutes from "date-fns/differenceInMinutes";
import sub from "date-fns/sub";
```

Reply:
I just did some tests and it looks like the unused functions are not included in the final bundle even when using the `import { sub } from "date-fns";` syntax.

Turn's out Nextjs has a [compile step](https://vercel.com/blog/how-we-optimized-package-imports-in-next-js) that automatically converts `import { sub } from "date-fns"` to `import sub from "date-fns/sub"`. This feature is configurable, but is enabled by default for the most common libraries including date-fns.

Reply:
Oh, that's cool. I mean I'm not hard on this. If this behaviour is not consistent across other envs I'd still use unified way. Again not a blocker from me

Reply:
One thing we could do is set the optimisation config explicitly in the `next.config.js` file to at least not depend on an unknown library list. Let's revisit this after the rebrand when the codebase will be more stable (lots of new libraries being added).